packages/pn-personafisica-login/src/pages/redirect/Redirect.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from "react";
+
+const RedirectPage = () => {
+  const redirectUrl = sessionStorage.getItem('redirectUrl');
+  const token = window.location.hash;
+
+  useEffect(() => {
+    if (redirectUrl) {
+      sessionStorage.clear();
```

Comment:
uniformiamo l'accesso al sessionStorage usando il metodo storageOpsBuilder in storage.utility.ts (vedi esempio in storage.ts)